docker/dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM 959744386191.dkr.ecr.eu-west-1.amazonaws.com/podpoint/php:7.1-dev
```

Comment:
I have been chatting with Martin about whether or not this link should be directly included in the dockerfile. As this file will also be included in the vendor folder for other projects and if it's going to be made public in packagist. It would expose this image url to the world. Not really a problem in itself as the image is not publicly accessible but just doesn't feel right or best practice.

The solution Martin recommended was adding a .env to this package and having the image url pulled from there. Which I think is a good idea if this package is going to be made public. If not them I think it's still a good idea but for an E&L. There also the problem of communicating that image url for any future dev that needs to set this package up. Not necessarily a problem unique to this package or setup because other projects already have this issue.

Reply:
could we use a public image? don't see a reason to use ours here